tests/unit/test_protobuf.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from jina.proto.jina_pb2 import Document
+from tests import JinaTestCase
+from google.protobuf.json_format import MessageToJson, Parse
+
+
+class ProtobufTestCase(JinaTestCase):
+
+    def test_tags(self):
+        d = Document()
+        d.tags['int'] = 1  # will convert to float!!!
+        d.tags['str'] = 'blah'
+        d.tags['float'] = 0.1234
+        d.tags['bool'] = True
+        d.tags['nested'] = {'bool': True}
```

Comment:
One suggestion is we can setup a set of test fixtures in tests and reuse these fixtures in other tests.

Reference:
> One great use for fixtures is to set up structures or variables the same way for all tests. This is to make sure that tests can run individually as well as a set and in any order. [link](https://pythontesting.net/framework/unittest/unittest-fixtures/)

Might be related to #692